source/examples/stream.cpp

```diff
@@ -2,15 +2,17 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-#include <CL/sycl.hpp>
+#include <sycl/sycl.hpp>
 
 int main() {
   sycl::queue q;
+
   q.submit([&](sycl::handler &h) {
     // setup sycl stream class to print standard output from
     // device code
     auto out = sycl::stream(1024, 768, h);
     auto task = [=]() { out << "In a task\n"; };
-    h.single_task(task);
+    h.single_task<class stream_task>(task);
```

Comment:
This is probably not part of standard sycl because it requires host compiler support. We need a way to verify that examples can be called with at least dpcpp/computecpp.

Reply:
I added dpcpp testing. We still need computecpp testing